src/main/python/pybuilder/plugins/python/coverage_plugin.py

```diff
@@ -39,6 +39,8 @@ def init_coverage_properties(project):
     project.build_depends_on("coverage")
 
     project.set_property_if_unset("coverage_threshold_warn", 70)
+    project.set_property_if_unset("coverage_branch_threshold_warn", 0)
```

Comment:
Maybe they want it but saw it was not possible and gave up. There's not a lot of visibility on the release notes so they might not notice. But maybe you're right and I'm thinking too much about this :-P